### PR TITLE
US-513232 Expose path and pathType for configured host in Ingress

### PR DESCRIPTION
**Pull request by MadhuriArugula**
Expose path and pathType for configured host in Ingress

### CODE REVIEW

File: charts/pega/templates/_pega-k8s-ingress.tpl [hunk 1]
```diff
@@ -32,7 +32,10 @@ spec:
         backend:
 {{ include "ingressServiceC11n" . | indent 10 }}
       {{ end }}
-      - pathType: ImplementationSpecific
+      - pathType: {{ include "hostPathType" . }}
```
Comment (kishorv10) [inline charts/pega/templates/_pega-k8s-ingress.tpl:35]: Can we templatize these 4 lines of code as well? Also, Add some test cases?

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Comment by MadhuriArugula** [comment]
@taz-pega-work Can you please review and provide approvals on this?

**Comment by pegaautomationuser** [comment]
Starting PR-496 validation on -> integ-eks

**Review by kishorv10 — APPROVED** [review]

**Review by arvasrikanth — APPROVED** [review]

**Pull request merged**